components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -7,12 +7,11 @@ use std::collections::HashSet;
 use std::slice::Iter;
 use std::time::{Duration, SystemTime};
 
+use rand::Rng;
```

Comment:
This is to follow this ordering rule of importing a crate:

1. std
2. 3rd
3. self